admin_guide/scheduler.adoc

```diff
@@ -367,3 +367,204 @@ Since the scheduler is a plug-in, it can be replaced in favor of an alternate
 implementation. The scheduler code has a clean separation that watches new pods
 as they get created and identifies the most suitable node to host them. It then
 creates bindings (pod to node bindings) for the pods using the master API.
+
+
+[[controlling-pod-placement]]
+== Controlling Pod Placement
+
+As a cluster administrator, you can set a policy to prevent application
+developers with certain roles from targeting specific nodes when scheduling
+pods. 
+
+[IMPORTANT]
+====
+This process involves the *pod/binding* permission
+link:../admin_guide/manage_authorization_policy.html[role], which is needed to
+target particular nodes. The constraint on the use of the `nodeSelector` field
+of a pod configuration is based on the *pods/binding* permission and the
+`nodeSelectorLabelBlacklist` configuration option.
+====
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in new
+`nodeSelector` fields of a pod configuration. Users, service accounts, and
+groups that have the *pods/binding* permission can specify any node selector.
+Those without the *pods/binding* permission are prohibited from setting a
+`nodeSelector` for any label that appears in `nodeSelectorLabelBlacklist`.
+
+For example, you might have a cluster consisting of five data centers spread
+across two regions. In the US, "us-east", "us-central", and "us-west"; and
+in APAC, "apac-east" and "apac-west". Each node in each geographical region
+link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[should be labeled
+accordingly]. For example, `region: us-east`.
+
+As a cluster administrator, you can create an infrastructure where application
```

Comment:
Maybe change the tone here to make it more clear that this is a hypothetical example, e.g. "As a cluster administrator, you might have an infrastructure where..."

I don't want people to read this and think "well here's how we would set up a geographic constraint for pod deployment!" :smiley: